query/src/sql/sql_common.rs

```diff
@@ -85,4 +85,12 @@ impl SQLCommon {
             ))),
         }
     }
+
+    pub fn short_sql(query: &str) -> String {
+        if query.len() >= 64 && query[..=6].eq_ignore_ascii_case("INSERT") {
```

Comment:
Maybe need trim for `.    INSERT INTO table()`

Reply:
I don't want to use parser/regexp here, so this it's the simple way to store short insert sqls.

Reply:
Okay.